src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -149,7 +149,7 @@ namespace ts.server.typingsInstaller {
             }
             const discoverTypingsResult = JsTyping.discoverTypings(
                 this.installTypingHost,
-                this.log.isEnabled() ? this.log.writeLine : undefined,
+                this.log.isEnabled() ? this.log.writeLine.bind(this.log) : undefined,

```

Comment:
Use `line => this.log.writeLine(line)` instead of `.bind`? AFAIK we'd avoided using `bind` in most places, why stop now?